build-scripts/components/containerd/build.sh

```diff
@@ -4,6 +4,7 @@ INSTALL="${1}/bin"
 mkdir -p "${INSTALL}"
 
 for bin in ctr containerd containerd-shim containerd-shim-runc-v1 containerd-shim-runc-v2; do
+  export SHIM_CGO_ENABLED=1
```

Comment:
While we are here, can we also set `STATIC=1` to get statically linked binaries?